packages/iov-bns/src/bnsconnection.posttxs.spec.ts

```diff
@@ -603,6 +612,275 @@ describe("BnsConnection (post txs)", () => {
       connection.disconnect();
     });
 
+    it("can register a domain", async () => {
+      pendingWithoutBnsd();
+      const connection = await BnsConnection.establish(bnsdTendermintUrl);
+      const registryChainId = connection.chainId();
```

Comment:
This chain ID getter recently changed from a function to a property (https://github.com/iov-one/iov-core/pull/1376). So it must be

```ts
const registryChainId = connection.chainId;
```